modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -25,7 +25,7 @@
  * @since 3.23.0
  */
 class Link_In_Bio extends Widget_Base {
-	const TAB_ADVANCED = 'advanced-tab-links';
+	const TAB_ADVANCED = 'advanced-tab-links-in-bio';
```

Comment:
@louiswol94 I know we changed this but the previous name broke the tab for some reason.